src/Devices/PHProbe.cpp

```diff
@@ -100,27 +136,36 @@ void PHProbe::setHighpointCalibration(float highpoint) {
   snprintf_P(buffer, sizeof(buffer), (PGM_P)F("Cal,High,%i.%03i\r"), (int)highpoint,
              (int)(highpoint * 1000 + 0.5) % 1000);
   Serial1.print(buffer);  // send that string to the Atlas Scientific product
+#if defined(ARDUINO_CI_COMPILATION_MOCKS)
+  this->calibrationPoints += 1;
+#endif
   serial(F("PHProbe::setHighpointCalibration(%i.%03i)"), (int)highpoint, (int)(highpoint * 1000) % 1000);
 }
 
 void PHProbe::setLowpointCalibration(float lowpoint) {
   char buffer[16];
   snprintf_P(buffer, sizeof(buffer), (PGM_P)F("Cal,low,%i.%03i\r"), (int)lowpoint, (int)(lowpoint * 1000 + 0.5) % 1000);
   Serial1.print(buffer);  // send that string to the Atlas Scientific product
+#if defined(ARDUINO_CI_COMPILATION_MOCKS)
+  this->calibrationPoints += 1;
+#endif
   serial(F("PHProbe::setLowpointCalibration(%i.%03i)"), (int)lowpoint, (int)(lowpoint * 1000) % 1000);
 }
 
 void PHProbe::setMidpointCalibration(float midpoint) {
   char buffer[16];
   snprintf_P(buffer, sizeof(buffer), (PGM_P)F("Cal,mid,%i.%03i\r"), (int)midpoint, (int)(midpoint * 1000 + 0.5) % 1000);
   Serial1.print(buffer);  // send that string to the Atlas Scientific product
+#if defined(ARDUINO_CI_COMPILATION_MOCKS)
+  this->calibrationPoints = 1;
+#endif
   serial(F("PHProbe::setMidpointCalibration(%i.%03i)"), (int)midpoint, (int)(midpoint * 1000) % 1000);
 }
 
 #if defined(ARDUINO_CI_COMPILATION_MOCKS)
```

Comment:
I see that you are setting the calibrationPoints earlier; I don't have the ability to search for senders so don't know how this method is used.